spec/routing/routing_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'routes', type: :routing do
+  describe 'objects/:id/file' do
+    it 'file_object_path' do
+      druid = 'druid:ab123cd4567'
+      expect(get: file_object_path(id: druid)).to route_to(controller: 'objects', action: 'file', id: druid)
+    end
+  end
```

Comment:
I wrote the routing spec when I was having trouble getting the request spec to work.  I can remove it if folks don't like it, or I can complete it, perhaps as part of issue #1243 (document API endpoints)